src/TaskEither.ts

```diff
@@ -438,6 +438,15 @@ export const chainTaskOptionK = <E>(
 ): (<A, B>(f: (a: A) => TaskOption<B>) => (ma: TaskEither<E, A>) => TaskEither<E, B>) =>
   flow(fromTaskOptionK(onNone), chain)
 
+/**
+ * @category combinators
+ */
```

Comment:
CI is failing because here we should add the `since` tag + version, i.e. `@since 2.12.3`